Ginger/GingerCoreCommon/UIElement/ElementInfo.cs

```diff
@@ -314,6 +461,28 @@ public object GetElementData(eLocateBy elementLocateBy = eLocateBy.ByXPath, stri
             return mData;
         }
 
+
+        public static bool IsTheSameElement(ElementInfo firstEI, ElementInfo secondEI)
+        {
```

Comment:
where IsTheSameElement is been used? should not be on ElementInfo class because might be implmented diffrently per platform

Reply:
Moved to POMAllElementspage